assets/css/list.css

```diff
@@ -0,0 +1,78 @@
+/* spaced list */
+
+.list-ordered-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.list-type-upper-alpha {
+  list-style-type: upper-alpha;
+}
+
+.list-type-lower-alpha {
+  list-style-type: lower-alpha;
+}
+
+.list-type-upper-roman {
+  list-style-type: upper-roman;
+}
+
+.list-type-lower-roman {
+  list-style-type: lower-roman;
+}
+
+ol {
+  font-size: 1.5rem;
+}
+
+.list-unordered-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+ul {
+  font-size: 1.5rem;
+}
```

Comment:
as a suggestion you should style directly to html. Instead  you can give class or id.